juju/juju
# Populate All Address Origins

## Description of change

Updates the common networking API so that for link-layer device updates:
- All device addresses have origin populated.
- Log messages use the _INFO_ level where appropriate.
- Log messages include the machine ID so that we can identify updates that are logged asynchronously.
- Some redundant test infrastructure is removed.

## QA steps

- Bootstrap LXD and add a machine.
- Connect to Mongo and run `db.ip.addresses.find().pretty()`. All addresses should include the "machine" origin.

## Documentation changes

None.

## Bug reference

N/A
